conf/database-connection-listener.env

```diff
@@ -0,0 +1,7 @@
+# This is the configuarion used by the vulnerability engine's
+# evaluator component to connect to the database.
+POSTGRESQL_DATABASE=vulnerability
+POSTGRESQL_HOST=ve_database
+POSTGRESQL_PORT=15432
```

Comment:
Port number 15432 is used only for outside connection. 
In the internal network of OS or docker should be 5432.

Reply:
I noticed that you made that change.  I will update the manager and new listener config to use port 5432.